src/models/token.model.js

```diff
@@ -0,0 +1,22 @@
+const { DataTypes } = require('sequelize');
+const { sequelize } = require('../utils/db');
+const { User } = require('./user.model');
+
+const Token = sequelize.define(
+  'token',
+  {
+    refreshToken: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+  },
+  {
+    createdAt: false,
+    updatedAt: false,
+  },
+);
+
+Token.belongsTo(User);
+User.hasOne(Token);
```

Comment:
The `User.hasOne(Token)` association suggests that each user can only have one token. If users can have multiple tokens, consider using `User.hasMany(Token)` instead to accurately represent the relationship.